lib/Rex/Virtualization/Lxc.pm

```diff
@@ -0,0 +1,49 @@
+#
+# (c) Oleg Hardt <litwol@litwol.com>
```

Comment:
Since this is code you wrote, you own copyright over it, so it's best ot indicate it as such :wink: